src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -126,12 +127,29 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                 );
                 String defaultsFilePath = DefaultUseCases.getDefaultsFileByUseCaseName(useCase);
                 useCaseDefaultsMap = ParseUtils.parseJsonFileToStringToStringMap("/" + defaultsFilePath);
-
-                if (request.hasContent()) {
+                List<String> requiredParams = DefaultUseCases.getRequiredParamsByUseCaseName(useCase);
+
+                if (request.hasContent() == false && requiredParams.size() != 0) {
```

Comment:
This would fail for the templates which doesn't have required params.

Reply:
Made a modification, tested with local bi-encoder which requires no parameters : 
```
curl -i -XPOST "http://localhost:9200/_plugins/_flow_framework/workflow?use_case=local_neural_sparse_search_bi_encoder&provision=true"
HTTP/1.1 201 Created
X-OpenSearch-Version: OpenSearch/3.0.0-SNAPSHOT (opensearch)
content-type: application/json; charset=UTF-8
content-length: 38

{"workflow_id":"CIbTzo4BDyZpM8DAIpBz"}%                                                                                                                                              

```